pages/glossary/valid-language-tag.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Valid Language Subtag
-key: valid-language-subtag
+title: Valid Language Tag
```

Comment:
```suggestion
title: Language Tag with valid primary subtag
```
Not convinced that it is good 🤔 But "valid language **tag**" is really confusing.

Reply:
I don't like that either. That almost inlines the entire phrase. It isn't a "valid primary subtag" either. We're not checking that it's "valid", we're checking that it exists in the registry.